src/pages/vaults/_components/commons/VaultHealthBar.tsx

```diff
@@ -68,9 +68,9 @@ export function VaultHealthBar (props: VaultHealthBarProps): JSX.Element {
         </div>
       </div>
       <div className='relative flex mt-2.5 items-center'>
-        <div className='w-full flex rounded-lg h-4 bg-gray-100 border overflow-hidden'>
+        <div className='w-full flex rounded-lg h-4 bg-gray-100 dark:bg-gray-500 border-gray-300 dark:border-gray-800 border overflow-hidden '>
```

Comment:
```suggestion
        <div className='w-full flex rounded-lg h-4 bg-gray-100 dark:bg-gray-700 border-gray-300 dark:border-gray-800 border overflow-hidden '>
```
let's follow LW? 
<img width="357" alt="image" src="https://user-images.githubusercontent.com/40191153/173331045-125d233a-f621-45cb-99e2-fe19a51f60ff.png">